lib/ansible/cli/scripts/ansible_connection_cli_stub.py

```diff
@@ -224,12 +224,16 @@ def shutdown(self):
 def main(args=None):
     """ Called to initiate the connect to the remote device
     """
-    parser = argparse.ArgumentParser(prog='ansible-connection', add_help=False)
-    parser.add_argument('--version', action=AnsibleVersion, nargs=0)
+
+    parser = opt_help.argparse.ArgumentParser(prog='ansible-connection', add_help=False)
+    opt_help.add_verbosity_options(parser)
```

Comment:
Does it break something to support --help?
```suggestion
    parser = opt_help.create_base_parser(prog='ansible-connection')
```
(which would also add --version back and fix entry_points test)

Reply:
`ansible-connection` isn't mean to be a human initiated CLI utility, so when I added `argparse` to support `--version` specifically for the `entry_points` test, I disabled `--help`.

Reply:
Okay, makes sense. I saw when you added it but wasn't sure since we did have tests calling it directly.